indigo_app/forms.py

```diff
@@ -242,7 +242,7 @@ class WorkFilterForm(forms.Form):
     q = forms.CharField()
     stub = forms.ChoiceField(choices=[('', 'Exclude stubs'), ('only', 'Only stubs'), ('all', 'Everything')])
     status = forms.MultipleChoiceField(choices=[('published', 'published'), ('draft', 'draft')])
-    subtype = forms.ModelChoiceField(queryset=Subtype.objects.all(), empty_label='All works')
+    subtype = forms.ChoiceField(choices=[('', 'All works'), ('act', 'Acts only')] + [(s.abbreviation, s.name) for s in Subtype.objects.all()])
```

Comment:
This is causing the build to fail. You can't access the database in your class declaration (the code needs to be "compiled" before the database is setup, and classes are established during this "compilation" phase).

Instead, either do it in `__init__` or create a function `subtype_choices` and use `choices=subtype_choices`.

Reply:
Thank you for spotting this